tests/lib/plugin.js

```diff
@@ -192,76 +211,76 @@ describe("plugin", () => {
             "var foo = blah",
             "```"
         ].join("\n");
-        const report = cli.executeOnText(code, "test.md");
-
-        assert.strictEqual(report.results[0].messages[0].message, "'baz' is not defined.");
-        assert.strictEqual(report.results[0].messages[0].line, 7);
-        assert.strictEqual(report.results[0].messages[0].endLine, 7);
-        assert.strictEqual(report.results[0].messages[1].message, "'blah' is not defined.");
-        assert.strictEqual(report.results[0].messages[1].line, 11);
-        assert.strictEqual(report.results[0].messages[1].endLine, 11);
+        const results = await eslint.lintText(code, { filePath: "test.md" });
+
+        assert.strictEqual(results[0].messages[0].message, "'baz' is not defined.");
+        assert.strictEqual(results[0].messages[0].line, 7);
+        assert.strictEqual(results[0].messages[0].endLine, 7);
+        assert.strictEqual(results[0].messages[1].message, "'blah' is not defined.");
+        assert.strictEqual(results[0].messages[1].line, 11);
+        assert.strictEqual(results[0].messages[1].endLine, 11);
     });
 
-    it("should run on .mkdn files", () => {
-        const report = cli.executeOnText(shortText, "test.mkdn");
+    it("should run on .mkdn files", async () => {
+        const results = await eslint.lintText(shortText, { filePath: "test.mkdnd" });
```

Comment:
Hopefully the last test failure is just due to a typo here from the API conversion 🤞

```suggestion
        const results = await eslint.lintText(shortText, { filePath: "test.mkdn" });
```